### PR TITLE
[FLINK-25705][docs]Translate "Metric Reporters" page of "Deployment" 

### PR DESCRIPTION
**Pull request by ChengkaiYang2022**
## What is the purpose of the change

Translate the "Metric Reporters" page of "Deployment".This pr is based on previous work of https://github.com/apache/flink/pull/19023.

## Brief change log

*(for example:)*
  - Translate some tables in this page .

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluster with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no
  - The serializers: (yes / no / don't know) no 
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) no

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by ChengkaiYang2022** [comment]
This pr is based on https://github.com/apache/flink/pull/19023.
hi, @RocMarshal  I translate the tables and was able to run the website on my laptop. Please take a look thanks!

**Comment by RocMarshal** [comment]
Cool~ @ChengkaiYang2022 . I'll check it ASAP.

**Review by healchow — APPROVED** [review]

**Comment by ChengkaiYang2022** [comment]
> Cool~ @ChengkaiYang2022 . I'll check it ASAP.

Hi，@RocMarshal any update on your review?

**Comment by RocMarshal** [comment]
@ChengkaiYang2022 sorry for the late check. Would you mind resolving the conflicts before the review work ? Thanks a lot.

**Comment by ChengkaiYang2022** [comment]
> @ChengkaiYang2022 sorry for the late check. Would you mind resolving the conflicts before the review work ? Thanks a lot.

@RocMarshal ,I see some pr discussions to use rebase command instead of merge command and squash commits before do that. So Is that Okay I follow these steps?
1. First, run `rebase -i Head~4`, to squash 4 commits in these PR into 1 commit and pust on my own remote branch.
2. In my local branch,run `git fetch origin master` and git `rebase origin/master` to solve the conflicts.

In the last [PR](https://github.com/apache/flink/pull/19023), I use the merge command and unable to squash the commits into one commit after merged.

Would you mind give me more details about how to solve the conflict, or like should we squash commits at the end of the review or at the very beginning of the review? 
I would like to have a clearer understanding of the community things, Thank you very much!

**Comment by RocMarshal** [comment]
Yes. Your description about the `squish` and `rebase` will work. Feel free go ahead.

**Comment by ChengkaiYang2022** [comment]
@RocMarshal, I resolved the conflicts, here is the
[link](http://localhost:1313/flink/flink-docs-master/zh/docs/deployment/metric_reporters/#%E5%8F%91%E9%80%81%E5%99%A8)
Screenshot 2022-05-03 at 01-06-12 Metric Reporters

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (RocMarshal) [inline docs/content.zh/docs/deployment/metric_reporters.md:37]:
```suggestion
下面列出了所有发送器都适用的参数，可以通过配置文件中的 `metrics.reporter.<reporter_name>.<property>` 项进行配置。有些发送器有自己特有的配置，详见该发送器章节下的具体说明。
```

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (RocMarshal) [inline docs/content.zh/docs/deployment/metric_reporters.md:201]:
```suggestion
| Histogram | Summary    | 分位数为 .5，.75，.95，.98，.99 和 .999。  |
```

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (RocMarshal) [inline docs/content.zh/docs/deployment/metric_reporters.md:262]:
```suggestion
使用 Datadog 时，Flink 运行指标中的任何变量，例如 `<host>`、`<job_name>`、 `<tm_id>`、 `<subtask_index>`、`<task_name>`、 `<operator_name>`，都会被当作 `host:localhost`、`job_name:myjobname` 这样的 tag 发送。
```

**Review by RocMarshal — APPROVED** [review]
Thanks @ChengkaiYang2022 for the update.
LGTM on the whole. 
Could you  @gaoyunhaii @wuchong help to check it ?

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (ChengkaiYang2022) [reply docs/content.zh/docs/deployment/metric_reporters.md]: Thanks for that! Done

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (ChengkaiYang2022) [reply docs/content.zh/docs/deployment/metric_reporters.md]: Done

File: docs/content.zh/docs/deployment/metric_reporters.md
Comment (ChengkaiYang2022) [reply docs/content.zh/docs/deployment/metric_reporters.md]: Done

**Comment by ChengkaiYang2022** [comment]
Could you @leonardBang help to check it ?

**Comment by gaoyunhaii** [comment]
@ChengkaiYang2022 Very thanks for the PR and sorry for the delay, I'll have a look

**Comment by ChengkaiYang2022** [comment]
> @ChengkaiYang2022 Very thanks for the PR and sorry for the delay, I'll have a look

Thanks!

**Comment by ChengkaiYang2022** [comment]
> @ChengkaiYang2022 Very thanks for the PR and sorry for the delay, I'll have a look

I understand that Flink's committer receives a lot of pending and review requests every day. 
And I might be pending the wrong committer cause I'm not sure which committer is familiar or focused on the metric part.
Also, thanks again for helping to review, and if there are any translation inaccuracies in the document, please let me know.

File: docs/layouts/shortcodes/generated/metric_reporters_section_zh.html
Comment (gaoyunhaii) [inline docs/layouts/shortcodes/generated/metric_reporters_section_zh.html:12]: This item seems removed from the English version?

**Review by gaoyunhaii — COMMENTED** [review]
Very thanks @ChengkaiYang2022 for the PR! I only have one minor comment to have a double check.

File: docs/layouts/shortcodes/generated/metric_reporters_section_zh.html
Comment (ChengkaiYang2022) [reply docs/layouts/shortcodes/generated/metric_reporters_section_zh.html]: Fixed!

**Review by gaoyunhaii — APPROVED** [review]
Thanks @ChengkaiYang2022 for the PR! LGTM, will merge~

**Pull request closed without merging**
